package.json

```diff
@@ -176,6 +176,7 @@
 		"next-query-params": "^4.1.0",
 		"next-sitemap": "^3.1.21",
 		"next-themes": "^0.2.1",
+		"oas-normalize": "^8.4.1",
```

Comment:
We use `oas-normalize` for schema parsing, validation, and de-referencing.

Reply:
This tool seems so cool! 😄